doc/cephfs/fstab.rst

```diff
@@ -44,4 +44,4 @@ Ensure you use the ID (e.g., ``admin``, not ``client.admin``). You can pass any
 See `Authentication`_ for details. 
 
 
-.. _Authentication: ../../rados/operations/authentication/
+.. _Authentication: ../../rados/operations/user-management/
```

Comment:
might want to update the place referencing this section as well to reflect the updated section name of the "Authentication" section.